nemo/deploy/nlp/query_llm.py

```diff
@@ -527,3 +527,85 @@ def query_llm_streaming(
                     yield sentences
                 else:
                     yield partial_result_dict["outputs"]
+
+
+class NemoQueryTRTLLMPytorch(NemoQueryLLMBase):
```

Comment:
All the classes here in this file -- `NemoQueryTRTLLMPytorch`, `NemoQueryLLM`, etc -- are very similar.

Maybe we should work out a way to unify them. CC @oyilmaz-nvidia 

Besides, I think we don't need the constructor here. Let's remove it -- perhaps everywhere here -- ok?

Reply:
WDYT @oyilmaz-nvidia?